packages/core/test/insured-bridge/AVM_BridgeDepositBox.js

```diff
@@ -0,0 +1,228 @@
+        return ev.newBridgeAdmin == user1;
+      });
+    });
+
+    it("Set minimum bridging delay", async () => {
+      // Bridging delay should be set initially to the correct value.
+      assert.equal(await depositBox.methods.minimumBridgingDelay().call(), minimumBridgingDelay);
+
+      // Trying to change bridging delay from non-cross-domain owner should fail.
+      assert(await didContractThrow(depositBox.methods.setMinimumBridgingDelay(55).send({ from: rando })));
+      // Calling from the correct cross domain aliased address should work.
+      const tx = await depositBox.methods.setMinimumBridgingDelay(55).send({ from: bridgeAdminCrossDomainAlias });
+      assert.equal(await depositBox.methods.minimumBridgingDelay().call(), 55);
+      await assertEventEmitted(tx, depositBox, "SetMinimumBridgingDelay", (ev) => {
+        return ev.newMinimumBridgingDelay == 55;
+      });
+    });
+
+    it("Whitelist collateral", async () => {
+      // Trying to whitelist tokens from something other than the l2MessengerImpersonator should fail.
+      assert(
+        await didContractThrow(
+          depositBox.methods.whitelistToken(l1TokenAddress, l2Token.options.address, bridgePool).send({ from: rando })
+        )
+      );
+
+      // Calling from the correct cross domain aliased address should work.
+      const tx = await depositBox.methods
+        .whitelistToken(l1TokenAddress, l2Token.options.address, bridgePool)
+        .send({ from: bridgeAdminCrossDomainAlias });
+      assert.equal(
+        (await depositBox.methods.whitelistedTokens(l2Token.options.address).call()).l1Token,
+        l1TokenAddress
+      );
+      const expectedLastBridgeTime = await timer.methods.getCurrentTime().call();
+      await assertEventEmitted(tx, depositBox, "WhitelistToken", (ev) => {
+        return (
+          ev.l1Token == l1TokenAddress &&
+          ev.l2Token == l2Token.options.address &&
+          ev.lastBridgeTime == expectedLastBridgeTime &&
+          ev.bridgePool == bridgePool
+        );
+      });
+    });
+
+    it("Disable deposits", async () => {
+      // Trying to disable tokens from something other than the l2MessengerImpersonator should fail.
+      assert(
+        await didContractThrow(
+          depositBox.methods.setEnableDeposits(l2Token.options.address, false).send({ from: rando })
+        )
+      );
+
+      // Calling from the correct cross domain aliased address should work.
+      const tx = await depositBox.methods
+        .setEnableDeposits(l2Token.options.address, false)
+        .send({ from: bridgeAdminCrossDomainAlias });
+      assert.equal((await depositBox.methods.whitelistedTokens(l2Token.options.address).call()).depositsEnabled, false);
+      await assertEventEmitted(tx, depositBox, "DepositsEnabled", (ev) => {
+        return ev.l2Token === l2Token.options.address && ev.depositsEnabled == false;
+      });
+    });
+  });
+
+  describe("Box bridging logic", () => {
```

Comment:
Do you plan to do the "Box deposit logic" in the next PR?

Reply:
nope! that's already in the `BridgeDepositBox` set of tests. we have three tests files now:
- `BridgeDepositBox` that tests box deposit logic and associated events (but nothing on admin or cross-domain calls)
- `OVM_BridgeDepositBox` that tests cross-chain admin and cross-chain token transfers on optimism (but nothing on deposit logic, as this is covered in the `BridgeDepositBox` set of tests)
- `AVM_BridgeDepositBox` which is the same as the previous except it tests on arbitrum

this pattern means we dont need to repeat any of the deposit logic tests :)

Reply:
ooo very nice!